portal-bridge/Cargo.toml

```diff
@@ -35,7 +35,7 @@ portalnet.workspace = true
 prometheus_exporter.workspace = true
 rand.workspace = true
 reqwest.workspace = true
-reqwest-middleware = { version = "0.3.3", features = ["json"] }
+reqwest-middleware = { version = "0.3.3", default-features = false, features = ["json", "rustls-tls"] }
```

Comment:
My suspicion was that adding a dependency on this library in #1448 was the cause, since it was added without these features that we use on the base `reqwest` dependency. But, I added them and it didn't fix the error that was caused when building the release. I'm not sure whether or not to leave these in...